src/resources/parser/texture/img.js

```diff
@@ -15,6 +16,7 @@ function ImgParser(registry, retryRequests) {
     // by default don't try cross-origin, because some browsers send different cookies (e.g. safari) if this is set.
     this.crossOrigin = registry.prefix ? 'anonymous' : null;
     this.retryRequests = !!retryRequests;
+    this.useImageBitmap = typeof ImageBitmap !== 'undefined' && /Firefox/.test( navigator.userAgent ) === false;
```

Comment:
This will prevent Firefox from uses its ImageBitmap support when it works as required presumably? We probably need a better 'test' to check for working ImageBitmap support.

Reply:
I tried changing the test to:
```javascript
createImageBitmap(new ImageData(1, 1), {
    premultiplyAlpha: 'none'
})
.then(function (imageData) {
    self.useImageBitmap = true;
})
.catch(function (err) {
    self.useImageBitmap = false;
});
```

but unfortunately the async nature of .catch means the flag is set only later on, by which point the loader has already started processing texture requests.

Reply:
Sorry, can you explain exactly what's broken in Firefox? 😄

Reply:
:D Firefox throws if you attempt to call createImageBitmap with an options object (which is the means of specifying flipY and premultiplyAlpha).

https://bugzilla.mozilla.org/show_bug.cgi?id=1367251

Reply:
So I was attempting to construct an object with options. Unfortunately such a call on Firefox fails asynchronously and so you find out the result after you've already started loading textures.

Reply:
OK, well, I'm all right with that UA check for now.

Reply:
But consider adding your voice to that Bugzilla thread, saying the PlayCanvas engine is integrating this and we don't want to blanket ban using ImageBitmap in FF.

Reply:
done